docs/expose-applications-using-loadbalancer-type-service.md

```diff
@@ -69,7 +69,17 @@ Request Body:
 ## Supported Features
 
 ### Service annotations
-TBD
+- loadbalancer.openstack.org/floating-network-id
+- loadbalancer.openstack.org/floating-subnet
+- loadbalancer.openstack.org/floating-subnet-id
+- loadbalancer.openstack.org/subnet-id
+- loadbalancer.openstack.org/port-id
+- loadbalancer.openstack.org/connection-limit
+- loadbalancer.openstack.org/keep-floatingip
+- loadbalancer.openstack.org/proxy-protocol
+- loadbalancer.openstack.org/keep-client-ip
```

Comment:
in my opinion `keep-client-ip` is not perfect word for this, because we already have like keep-floatingip which means that it does not delete floatingip after lb is deleted. I suggest that we could use something like `add-forwarded-for`. Also people should understand that this does not work when using https at all. So should it be like something `http-forwarded-for`

Reply:
Good suggestion :-) will change this.